### ISSUE TITLE
ci-trial #2 failed

### ISSUE THREAD

**Issue by biaomingzhong** [biaomingzhong/ci-trial#1]
Build 'ci-trial' is failing!

Last 50 lines of build output:

```
Started by user mooyoo
Building in workspace /Users/Shared/Jenkins/Home/workspace/ci-trial
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/biaomingzhong/ci-trial.git # timeout=10
Fetching upstream changes from https://github.com/biaomingzhong/ci-trial.git
 > git --version # timeout=10
using GIT_ASKPASS to set credentials android developer
 > git fetch --tags --progress https://github.com/biaomingzhong/ci-trial.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 7bab873f632b91b71e5a24050b66d78e60661c1d (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7bab873f632b91b71e5a24050b66d78e60661c1d
Commit message: "添加新的行测试错误的情况"
 > git rev-list 3f0ccf74558b4ef83571c8b5937b216ef5cbd2ba # timeout=10
[ci-trial] $ /bin/sh -xe /Users/Shared/Jenkins/tmp/jenkins7112974130198016493.sh
+ chmod 744 ./jenkins/test.sh
[Set GitHub commit status (universal)] PENDING on repos [GHRepository@905222[description=<null>,homepage=<null>,name=ci-trial,license=<null>,fork=false,size=0,milestones={},language=Groovy,commits={},source=<null>,parent=<null>,responseHeaderFields={null=[HTTP/1.1 200 OK], Access-Control-Allow-Origin=[*], Access-Control-Expose-Headers=[ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval], Cache-Control=[private, max-age=60, s-maxage=60], Content-Encoding=[gzip], Content-Security-Policy=[default-src 'none'], Content-Type=[application/json; charset=utf-8], Date=[Thu, 16 Nov 2017 10:33:25 GMT], ETag=[W/"b4a6134b3f0588b45e6852499254e8c8"], Last-Modified=[Thu, 16 Nov 2017 10:28:08 GMT], OkHttp-Received-Millis=[1510828405625], OkHttp-Response-Source=[CACHE 200], OkHttp-Selected-Protocol=[http/1.1], OkHttp-Sent-Millis=[1510828405098], Server=[GitHub.com], Status=[200 OK], Strict-Transport-Security=[max-age=31536000; includeSubdomains; preload], Transfer-Encoding=[chunked], Vary=[Accept, Authorization, Cookie, X-GitHub-OTP, Accept-Encoding], X-Accepted-OAuth-Scopes=[repo], X-Content-Type-Options=[nosniff], X-Frame-Options=[deny], X-GitHub-Media-Type=[github.v3; format=json], X-GitHub-Request-Id=[F67D:23595:76DECE:9D8890:5A0D6975], X-OAuth-Scopes=[admin:repo_hook, gist, repo], X-RateLimit-Limit=[5000], X-RateLimit-Remaining=[4999], X-RateLimit-Reset=[1510832005], X-Runtime-rack=[0.058955], X-XSS-Protection=[1; mode=block]},url=https://api.github.com/repos/biaomingzhong/ci-trial,id=110957476]] (sha:7bab873) with context:Jenkins
Setting commit status on GitHub for https://github.com/biaomingzhong/ci-trial/commit/7bab873f632b91b71e5a24050b66d78e60661c1d
[ci-trial] $ /bin/sh -xe /Users/Shared/Jenkins/tmp/jenkins5411515712838449762.sh
+ ./jenkins/test.sh failed
build success
develop 1.0.1 add line
jenkins env var change author  branch name  job name ci-trial
develop 1.0.2 add new line
develop 1.0.3 new task
develop 1.0.4 something went wrong
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [quinlanzhong] 7bab873f632b91b71e5a24050b66d78e60661c1d - 添加新的行测试错误的情况

[View full output](http://172.16.20.252:8080/job/ci-trial/2/)
